### DISCUSSION TITLE
A little bit more detail in the error logs

### DISCUSSION THREAD

**Discussion by wolispace** [solspace/craft-freeform#1497] [Feature Requests]
Hi,

After upgrading to freeform 5 we are seeing mysterious errors like this:

`Form field was not an instance of FieldInterface {"field":null} []`

I'm not sure what a 'FieldInterface' is.
I have no idea which form has the field that has this problem.
I also have no idea which field has this problem.

Is it possible to include in each error:
1) The form name, ID or handle in the error logs 
2) The field name, ID or handle if known.

Thanks
Wallace

**Comment by kjmartens** [comment]
Just thought I'd provide an update on this one... In later versions of Freeform 5.9.x we greatly improved error and debug logging to include a lot more data where possible. 🙂

**Comment by wolispace** [comment]
This is excellent. Can't wait to get an error and see the improvements. Thanks.
